doc/source/examining/Loading_Spherical_Data.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Loading Spherical Data\n",
     "\n",
-    "With version 3.0 of yt, it has gained the ability to load data from non-Cartesian systems.  This support is still being extended, but here is an example of how to load spherical data from a regularly-spaced grid.  For irregularly spaced grids, a similar setup can be used, but the `load_hexahedral_mesh` method will have to be used instead.\n",
+    "With version 3.0 of yt, it has gained the ability to load data from non-Cartesian systems.  This support is still being extended, but here is an example of how to load spherical data from a regularly-spaced grid. For irregularly spaced grids, a similar setup can be used, either by using supplying the `cell_widths` parameter to `load_uniform_grid` (see [Stretched Grid Data](https://yt-project.org/docs/dev/examining/loading_data.html#stretched-grid-data) for more details), or by using the `load_hexahedral_mesh` method.\n",
```

Comment:
I'm wondering if it's still worth mentioning that yt<3 didn't have support for non-cartesian.

Reply:
Fixed